### PR TITLE
add option to skip compilation if no files in gulp stream

### PR DESCRIPTION
**Pull request by brendankenny**
addresses #3

feel free to bikeshed on the option name :)

### CODE REVIEW

File: test/gulp.js [hunk 1]
```diff
@@ -260,6 +260,43 @@ describe('gulp-google-closure-compiler', function() {
...
+          .pipe(assert.length(1))
+          .pipe(assert.first(function(f) {
+            f.contents.toString().should.eql('function log(a){console.log(a)}log("one.js");log("two.js");\n');
+          }))
+          .pipe(assert.end(done));
+    });
+
+    it('should generate no output without gulp.src files when stream input is required', function(done) {
+      this.timeout(30000);
+      this.slow(10000);
```
Comment (ChadKillingsworth) [inline test/gulp.js]: Minor issue - but this case should be extremely fast. I think the default is 250/750 ms.

**Comment by ChadKillingsworth** [comment]
I'm horrible at naming. I just suggested that one off the top of my head.

File: test/gulp.js (hunk 1 above)
Comment (brendankenny) [inline test/gulp.js]: good point, updated

**Comment by brendankenny** [comment]
yeah, I think it works. The only confusing part might be the buffer vs streaming concepts for gulp pipelines, but they're still streams even when streaming isn't supported, so...

Really it probably doesn't matter as long as it's documented.

**Comment by ChadKillingsworth** [comment]
The only people I've found who correctly know what the definition of a true gulp streams is, are plugin authors. It's pretty telling that the vast majority of gulp plugins seem to only support buffers. Stream support in this plugin is pointless as long as the sourcemaps plugin only supports buffers.

**Comment by ChadKillingsworth** [comment]
I just published the package as the official release - no longer under the "beta" tag.

**Pull request merged**